security/pkg/nodeagent/sds/sdsservice_test.go

```diff
@@ -291,6 +292,14 @@ func TestStreamSecretsPush(t *testing.T) {
 
 	verifySDSSResponse(t, resp, fakePushPrivateKey, fakePushCertificateChain)
 
+	key := cache.ConnKey{
+		ProxyID:      proxyID,
+		ResourceName: req.ResourceNames[0],
+	}
+	if _, ok := st.secrets.Load(key); ok != true {
+		t.Errorf("failed to find cached secret")
```

Comment:
nit: s/failed/Failed

Reply:
error message should start with lowercase, to align with rest of code

Reply:
Okay, there are a couple of other places where it's capitalized: See https://github.com/istio/istio/pull/11160/files/9ce00ddeb28f956fdf3f4a3180b01655f779cea8#diff-5230137c1e069ce6fd20fab17cbf88acR204.